.github/workflows/jit.yml

```
@@ -1,10 +1,19 @@
 name: JIT
 on:
   pull_request:
-    paths: '**jit**'
+    paths:
+    - '**/*jit*/**'
+    - '**/*jit*.*'
```

```suggestion
      - '**jit**'
```

(And adjust spacing to match the style further down the file)